content/2. Ondersteunende-informatie/4. Techniek/Git/3. Branches.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 3. Branches
+draft: false
+---
+
+> Meer informatie over branches: [Git branches](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell)
+## Wat zijn branches
+Een branch is een afzonderlijke lijn van ontwikkeling in een Git-repository. Bij het aanmaken van een repository komt altijd de branch `main` of `master` als hoofdbranch. Naast deze hoofdbranches kan je zelf ook branches erbij maken, zodat je kan werken aan de code zonder de hoofdcode te beïnvloeden.
+
+In een repository voor een codeer project is het gebruikelijk om in ieder geval een `main` en een `dev`/`development` branch te hebben. Bij een project kan dan elke nieuwe feature een eigen branch krijgen als voorbeeld.
+
+## Branches toevoegen
+Binnen een repository waar jij de rechten hebt om veranderingen te maken aan de repository, kan jij nieuwe branches aanmaken. Een nieuwe branch moet altijd gebaseerd worden op een andere branch (vaak de `main` of `development` branch).
+
+### Lokale en remote branches
+Branches kunnen local of remote zijn. 
+Lokale branches zijn, zoals de naam al zegt, op jouw machine aanwezig, deze zijn (nog) niet zichtbaar voor andere.
+
+[Remote branches](https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches) werken net iets anders. In Git verwijst een remote branch naar een branch die bestaat op een externe repository in plaats van in jouw lokale repository. Remote branches fungeren als aanwijzers naar de status van branches in de externe repository. Je kunt deze branches lokaal bijhouden om op de hoogte te blijven van wijzigingen van andere bijdragers.
+
+Een remote branch kan je identificeren aan het feit dat het in de meeste gevallen `origin/[BRANCH_NAAM]` zal heten. `origin` Is de standaard instelling, deze kan zelf een andere naam krijgen door het clone commando iets anders uit te voeren
+```
+git clone -o booyah
+```
+- Hiermee wordt `origin` vervangen met `booyah`.
```

Comment:
Misschien beter om hier een formeler alternatief voor te gebruiken. Iets als `windesheim` bijvoorbeeld. Ook handig om specifiek te benoemen wat de `-o` dan precies doet. Veranderd het alleen `origin` of nog meer